inst/rmarkdown/templates/wellcomeor/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,196 @@
+mean. Then as $n$ approaches infinity, the random variables
+$\sqrt{n}(S_n - \mu)$ converge in distribution to a normal
+$\mathcal{N}(0, \sigma^2)$.
+
+## Lists
+
+You can make lists with automatic numbering \dots
+
+1.  Like this,
+2.  and like this.
+
+\dots or bullet points \dots
+
+-   Like this,
+-   and like this.
+
+# Methods
+
+Methods should include a brief discussion of allowances made (if any) for
+controlling bias or unwanted sources of variability, and the limitations of the
+datasets.
+
+# Results
+
+This section is not essential for Web Tool papers.
+
+# Discussion
+
+The discussion should include the implications of the article results in view of
+prior work in this field.
+
+# Conclusions
+
+Please state what you think are the main conclusions that can be realistically
+drawn from the findings in the paper, taking care not to make claims that cannot
+be supported.
+
+## Author contributions
+
+In order to give appropriate credit to each author of an article, the individual
+contributions of each author to the manuscript should be detailed in this
+section. We recommend using author initials and then stating briefly how they
+contributed.
+
+## Competing interests
+
+All financial, personal, or professional competing interests for any of the
+authors that could be construed to unduly influence the content of the article
+must be disclosed and will be displayed alongside the article.
+
+## Grant information
+
+Please state who funded the work discussed in this article, whether it is your
+employer, a grant funder etc. Please do not list funding that you have that is
+not relevant to this specific piece of research. For each funder, please state
+the funder's name, the grant number where applicable, and the individual to whom
+the grant was assigned.
+
+## Acknowledgements
+
+This section should acknowledge anyone who contributed to the research or the
+article but who does not qualify as an author based on the criteria provided
+earlier (e.g. someone or an organisation that provided writing assistance).
+Please state how they contributed; authors should obtain permission to
+acknowledge from all those mentioned in the Acknowledgements section.
+
+Please do not list grant funding in this section.
+
+# References
+
+References can be listed in any standard referencing style that uses a numbering
+system(i.e. not Harvard referencing style), and should be consistent between
+references within a given article.
+
+Reference management systems such as Zotero provide options for exporting
+bibliographies as Bib\TeX{} files. Bib\TeX{} is a bibliographic tool that is
+used with \LaTeX{} to help organize the user's references and create a
+bibliography. This template contains an example of such a file,
+\texttt{sample.bib}, which can be replaced with your own. You can \LaTeX{}
+command  \verb|\cite| to create in-text citationsor by using
+RMarkdown/Bookdown's citations [@Smith:2012qr], and [@Smith:2013jd]
```

Comment:
By using a section References in the example there will be two sections references in the PDF Output

Probably need to change the title here ? 
![image](https://user-images.githubusercontent.com/6791940/143575423-6836ef61-aa62-4199-8895-a5cb559648f3.png)

Reply:
Changed the title of this section in https://github.com/rstudio/rticles/pull/436/commits/d2a84aee3a42ed3ca43085011f61702c1603f0eb